eth/backend.go

```diff
@@ -374,6 +375,16 @@ func (s *Ethereum) APIs() []rpc.API {
 	// Append any APIs exposed explicitly by the consensus engine
 	apis = append(apis, s.engine.APIs(s.BlockChain())...)
 
+	// Append any Sequencer APIs as enabled
+	if s.config.RollupSequencerEnableTxConditional {
+		// an effort to detect if running in a non-sequencer mode
+		if len(s.config.RollupSequencerHTTP) > 0 {
+			log.Warn("SendRawTransactionConditional enabled when RollupSequencerHTTP is defined. Skipping...")
```

Comment:
Good q, curious to get your thoughts

So in the current implementation, it does **not** support have support for forwarding `eth_sendRawTranscationConditional` requests to the sequencer after validation. The caller needs to send the request directly to the sequencer. So this code block here was an effort to not add the api to the node if it wasn't the sequencer

Looking at this again though since we already have `RollupSequencerHTTP` maybe it's worth adding support for forwarding these requests as it's a small lift in the code. Note it's not the same as forwarding txs since we want to explicitly call `sendRawTransactionConditional` and not `sendRawTransaction` which would strip the conditional

Reply:
updated the implementation such that conditional txs can be forwarded to the sequencer if the `seqRPCService` is set.

This is the same client used by `SendTx` so we won't end up in a situation where the conditional is lost and wrongly propagated to the sequencer. & if it was a different client from `SendTx`, the clients are only created when `RollupSequencerHTTP` is set adding another layer of protection here

Reply:
LMK how the updated code looks